### DISCUSSION TITLE
Upgrading from Ventura to Sonoma broke my Plash

### DISCUSSION THREAD

**Discussion by Andreas-T** [sindresorhus/Plash#171] [Q&A]
Hi.
I use Plash for a simple local website that has worked for years without issues. I upgraded to Sonoma and I get a repeated prompt from Plash that it needs access to the folder where the website resides. I confirm and I get an error that the file could not be opened. Strangely, the website stills displays on the desktop. I've tried to give Plash full disk access but it made no difference. I also tried removing the website and adding it but now it won't add and gives the same error and thus is no longer working. I have version 2.15 from the App store. Any advice?

**Comment by sindresorhus** [comment]
See: https://github.com/sindresorhus/Plash/issues/168

**Comment by Andreas-T** [comment]
#168  is about Sequoia, I am talking about Sonoma and it's not a crash.
